test/e2e/index.ts

```diff
@@ -667,32 +647,40 @@ test(
     });
     const agent = await admin.generateAgentPubKey();
 
+    const zippedDnaBundle = fs.readFileSync("test/e2e/fixture/test.dna");
+
+    const appBundle: AppBundle = {
+      manifest: {
+        manifest_version: "1",
+        name: "app",
+        roles: [
+          {
+            name: role_name,
+            provisioning: {
+              strategy: CellProvisioningStrategy.Create,
+              deferred: false,
+            },
+            dna: {
+              bundled: "dna_1",
+              modifiers: { network_seed: "some_seed" },
+            },
+          },
+        ],
+        membrane_proofs_deferred: true,
+      },
+      resources: {
+        dna_1: zippedDnaBundle,
+      },
+    };
+
+    const zippedAppBundle = zlib.gzipSync(encode(appBundle));
```

Comment:
Why is this gzipped and not just encoded?

Reply:
Because it expects the bytes of a .happ file which is gzipped. This is the error returned without it:
```
Test caught exception:  internal_error: Conductor returned an error while using a ConductorApi: AppBundleError(MrBundleError(StdIoError(Custom { kind: InvalidInput, error: "invalid gzip header" })))
```

Reply:
https://github.com/holochain/holochain/blob/15cdd4bf429d0d0dcc02d03a18b9e732a35b066b/crates/mr_bundle/src/encoding.rs#L10

Reply:
Right, so `bytes` is not exact.